server/ot_rpc.go

```diff
@@ -73,23 +77,40 @@ func runTranscoder(n *core.LivepeerNode, orchAddr string, capacity int) error {
 		}
 		return err
 	}
+	exitc := make(chan os.Signal)
```

Comment:
Maybe add a comment for this part, "Catch interrupt signal to shut down transcoder"

Reply:
Fixed